### DISCUSSION TITLE
What's the preferred Official integration for Native App

### DISCUSSION THREAD

**Discussion by felipetesc** [emilk/egui#2057] [Q&A]
Hello there!!! I wanted to know what's the preferred integration for performatic native apps. I will use Windows 11, and in the future I intend to port it for Mac. I won't need Web backend. Thanks!

**Comment by ThePerfectComputer** [comment]
I'm not sure I understand what your question means? Egui is in some sense platform agnostic as far as I can tell?

I've tested egui code on MacOS, Windows, and Linux - all of which seem to run egui code just fine..

**Reply by felipetesc** [reply]
On the first page of this repository there is a list with the **Official integrations**:
egui_glium
egui_glow
egui-wgpu
